docs/Actions/send-email.md

```diff
@@ -0,0 +1,564 @@
+| Headers                       | Appends custom headers to the email.                                                                                                                                                                                                                                      | Yes             | Unset                                    |
+| Attach Portal Files           | Picks existing files from the system to send as attachments.                                                                                                                                                                                                              | No              | Unset                                    |
+| Attach Field #                | Form only parameters. Provides the ability to add up to 5 Single File Uploads fields to be sent as attachments.                                                                                                                                                           | No              | Unset                                    |
+| Attach from Token             | Computes file path from token. It must be relative to the portal home folder. Multiple items must be separated by semicolons.                                                                                                                                             | Yes             | `empty string`                           |
+| Ignore Errors                 | If errors are ignored, the submission of the form continues even when there is an error sending the email. Otherwise, an error message will be displayed to the user. Note that if the email is being sent async or is placed in a pickup folder, errors won't be caught. | No              | No                                       |
+
+## `Formats`
+
+| Format                                   | Details                                                                                                                                                                                                                      |
+| ---------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Html (And replace newline with `<br />`) | To simplify the input of the email we are replacing the newlines in the `Body` of the email with html tag `<br>`. While it is easier to compose the email it may have strange behaviors with some email clients like Outlook |

```

Comment:
```suggestion
| HTML (And replace newline with `<br />`) | To simplify the input of the email we are replacing the newlines in the `Body` of the email with the HTML tag `<br>`. While it is easier to compose the email it may have strange behaviors with some email clients like Outlook |
```